en/debug/jtag_pins.md

```diff
@@ -0,0 +1,130 @@
+nRESET | I/O | Target CPU reset signal. Typically connected to the RESET pin of the target CPU, which is typically called `nRST`.
+TRACECLK | Input | Input trace clock. Trace clock = 1/2 CPU clock.
+TRACEDATA[0] | Input | Input Trace data pin 0 (called TRACED0 in Pixhawk FMUv5x)
+
+>  **Note** The information above is reproduced from [J-Link / J-Trace User Guide: UM08001 > 18.1.2 Pinout for SW Mode](https://www.segger.com/downloads/jlink/UM08001) (Software Version: 6.54).
+
+
+## Pixhawk Series
+
+[Pixhawk series](https://docs.px4.io/master/en/flight_controller/pixhawk_series.html) boards provide access to JTAG through the **Pixhawk Debug Port**.
+The physical connector and pin order for this port varies across Pixhawk FMU versions.
+
+> **Note** The **Pixhawk Debug Port** also exposes a UART for access to the [System Console](../debug/system_console.md) (a console that provides low-level access to the system, debug output and analysis of the system boot process).
+
+<!--
+### FMUv2
+
+Does this have a debug port? where is spec
+Is it a Hirose DF13
+
+
+### FMUv3
+
+Does this have a debug port? where is spec
+Is it a Hirose DF13
+-->
+
+### FMUv4/FMUv5
+
+FMU V4,5 has a 6 pin JST GH
+
+[Pixracer](http://docs.px4.io/master/en/flight_controller/pixracer.html#pinouts): [Pixhawk debug port](https://pixhawk.org/pixhawk-connector-standard/#dronecode_debug) (JST SM06B connector)
+
+Pin | Signal | Volt
+--- | --- | ---
+1 (red) | VCC TARGET SHIFT | +3.3V
+2 (blk) | CONSOLE TX (OUT) | +3.3V
+3 (blk) | CONSOLE RX (IN) | +3.3V
+4 (blk) | SWDIO | +3.3V
+5 (blk) | SWCLK | +3.3V
+6 (blk) | GND | GND
+
+
+### FMUv5X
+
+FMUv5X will have a [10-pin JST SUR](https://order.jst-mfg.com/InternetShop/app/index.php?back=2&product=53574000&jgcd=0#showProductDetail).
+It adds JTAG pins: SWO, nRST. TRACECLK, TRACED0 Pins 7 and 8 are GPIOS to gate timing or 1 wire trace (full trace is on).
+<!-- is this link/example correct type for connector? -->
+
+FMU Debug Port Pinout
+
+Pin | Signal | Volt
+--- | --- | ---
+1 (red) | FMU VDD | +3.3V
+2 (blk) | CONSOLE TX (OUT) | +3.3V
+3 (blk) | CONSOLE RX (IN) | +3.3V
+4 (blk) | SWDIO | +3.3V
+5 (blk) | SWCLK | +3.3V
+6 (blk) | SWO | +3.3V
+7 (blk) | TRACECLK | +3.3V
+8 (blk) | TRACED0 | +3.3V
+9 (blk) | nRST | +3.3V
+10 (blk) | GND | GND
+
+
+> **Note** This board will have an additional 8-pin JST SUR for ARM ETM TRACE).
+
+
+## Hex Cube
+
+Cube debug port information can be found here: [Debug ports](http://docs.px4.io/master/en/flight_controller/pixhawk-2.html#debug-ports).
+
+If pulled apart the Cube has a 6 pin JST SUR connector (marked as J10) for both FMU and IO debugging.
+
+<!--
+Does the numbering match all the other 6 pin port examples.
+What label is the IO port marked as?
+So obviously it is much easier to just connect to the carrier board port. Who would know what interface that is. PhilipR?
+-->
+
+
+## Kakute F7
+
+There are Test points on the board.
```

Comment:
@davids5. So you're saying that the board does support this. NOthing shows up on an internet serach. Any ideas about how I find out what pads do SWO etc?

Reply:
Ask @bkueng for Photo I do not have one. @jinger26 can I get one?

Reply:
I'm not aware that the KakuteF7 exposes JTAG.
The board with pics is here: https://shop.holybro.com/kakute-f7_p1104.html (I can make one if there's anything specific).

Reply:
SWDIO -> TP3
SWCLK -> TP2
@bkueng can you see them and highlight in pic?

Reply:
Pin 72 on CPU -> TP2->SWCLK
Pin 76 on CPU -> TP3->SWDIO
![image](https://user-images.githubusercontent.com/1945821/69153880-da782a80-0a93-11ea-9b7f-a94f03af6f31.png)

Reply:
I'm assuming @bkueng is still going to be marking this up.

Reply:
Ah, magic. Thanks David.
![kakutef7_swdio](https://user-images.githubusercontent.com/281593/69228360-b5fb7b80-0b83-11ea-9db7-3d6e6a95fcd8.png)

Reply:
@bkueng Nice! We should also show GND   and the VDD_3V3 
![image](https://user-images.githubusercontent.com/1945821/69239146-71a1b880-0b4e-11ea-9d31-8746a5614ada.png)